### ISSUE TITLE
Explore ViT Vision Transformer Models

### ISSUE THREAD

**Issue by sbatchelder** [WHOIGit/image-classifier-dojo#7]
ViTs are architecturally different from CNNs in that they tokenize images and instead of performing convolutions.

The training requirements for batch-sizes and loss calculation and hyperparameters are different from CNNs so care will need to be taken.
